src/main.py

```diff
@@ -63,9 +63,18 @@ def mensa(update, context):
     if not resp.ok:
         context.bot.send_message(chat_id=update.message.chat_id, text="I failed miserably. Disgrace!")
         return
+
     jsonData = json.loads(resp.content)
     for elem in jsonData:
-        context.bot.send_message(chat_id=update.message.chat_id, text=elem["name"])
+        list=elem["notes"]
+        isVeg = 0
+        for veg in list:
```

Comment:
There is no need to loop over the elements of the `note` list. You can check via the mebership operator `in` - https://docs.python.org/3/reference/expressions.html#membership-test-operations.

Also note, that `veg` as an identifier is missleading.